beacon-chain/rpc/apimiddleware/custom_hooks.go

```diff
@@ -17,12 +17,12 @@ func wrapAttestationsArray(endpoint gateway.Endpoint, _ http.ResponseWriter, req
 	if _, ok := endpoint.PostRequest.(*submitAttestationRequestJson); ok {
 		atts := make([]*attestationJson, 0)
 		if err := json.NewDecoder(req.Body).Decode(&atts); err != nil {
-			return gateway.InternalServerErrorWithMessage(err, "could not decode attestations array")
+			return gateway.InternalServerErrorWithMessage(err, "could not decode body")
```

Comment:
I simplified some messages so that creating a new wrapper function is not vulnerable to typos. I had already made one when copying the implementation.